test-data/unit/check-generics.test

```diff
@@ -978,6 +978,30 @@ Bad = A[int] # type: ignore
 reveal_type(Bad) # E: Revealed type is 'Any'
 [out]
 
+[case testNoSubscriptionOfBuiltinAliases]
+from typing import List, TypeVar
+
+list[int]() # E: "list" is not subscriptable, use "typing.List" instead
```

Comment:
This is somewhat sad, because if you follow the instructions in the message, you get `List[int]()` which elicits a different error ("Type List cannot be instantiated; use list() instead"), but at runtime.